samples/storage_csm_powerflex_v2110.yaml

```diff
@@ -18,6 +18,9 @@ spec:
       #   false: disable storage capacity tracking
       storageCapacity: true
     configVersion: v2.11.0
+    # Controller count
+    # Allowed values: n, where n > 0
+    # Default value: None
```

Comment:
Same comment as above applies to all files

Reply:
updated